cpp/deplex/src/deplex/histogram.cpp

```diff
@@ -0,0 +1,51 @@
+#include "histogram.h"
+
+namespace deplex {
+Histogram::Histogram(int32_t nr_bins_per_coord, Eigen::MatrixXd const& normals, std::bitset<BITSET_SIZE> const& mask)
```

Comment:
Why do you call it normal if actually that is spherical coordinates passing here?
Also, let's move calculation of spherical coordinates here, passing normals as argument